src/foundry/foundry.js/clientDocuments/actor.d.ts

```diff
@@ -25,10 +26,258 @@ declare global {
    * ```
    */
   class Actor extends ClientDocumentMixin(foundry.documents.BaseActor) {
+    /**
+     * @override
+     * @param data    - Initial data provided to construct the Actor document
+     * @param context - The document context, see {@link foundry.abstract.Document}
+     */
+    constructor(
+      data: ConstructorParameters<typeof foundry.documents.BaseActor>[0],
+      context: ConstructorParameters<typeof foundry.documents.BaseActor>[1]
+    );
+
+    /**
+     * An object that tracks which tracks the changes to the data model which were applied by active effects
+     * @defaultValue `{}`
+     */
+    overrides: Record<string, unknown>;
+
+    /**
+     * A cached array of image paths which can be used for this Actor's token.
+     * Null if the list has not yet been populated.
+     * @defaultValue `null`
+     */
+    protected _tokenImages: string[] | null;
+
+    /**
+     * Cache the last drawn wildcard token to avoid repeat draws
+     * @defaultValue `null`
+     */
+    protected _lastWildcard: string | null;
+
     /**
      * A convenient reference to the file path of the Actor's profile image
      */
     get img(): this['data']['img'];
+
+    /**
+     * Provide an object which organizes all embedded Item instances by their type
+     */
+    get itemTypes(): Record<string, InstanceType<ConfiguredDocumentClass<typeof foundry.documents.BaseItem>>>;
+
+    /**
+     * Test whether an Actor entity is a synthetic representation of a Token (if true) or a full Entity (if false)
+     */
+    get isToken(): boolean;
+
+    /**
+     * An array of ActiveEffect instances which are present on the Actor which have a limited duration.
+     */
+    get temporaryEffects(): InstanceType<ConfiguredDocumentClass<typeof foundry.documents.BaseActiveEffect>>[];
```

Comment:
(Optional) suggestion for simplification:
```suggestion
    get temporaryEffects(): ReturnType<this['effects']['filter']>;
```